db/multi.go

```diff
@@ -0,0 +1,105 @@
+package db
+
+import (
+	"fmt"
+	"strings"
+)
+
+// MultiInserter makes it easy to construct a
+// `INSERT INTO table (...) VALUES ... RETURNING id;`
+// query which inserts multiple rows into the same table. It can also execute
+// the resulting query.
+type MultiInserter struct {
```

Comment:
Nit. Should only end in _er_ if it's an interface. I could take or leave this 😅

Reply:
or if it comes from the Interface region of Switzerland, otherwise it's just a sparkling structure

Reply:
I'm not a fan of the semantics of `db.MultiInsert.Add()`, but I'm totally open to other ideas for what to name this type. I hate "MultiInserter", I just kinda hate "MultiInsert" too :P